release-common-lib/src/main/java/org/reactome/util/ensembl/EnsemblServiceResponseProcessor.java

```diff
@@ -153,160 +150,249 @@ public EnsemblServiceResponseProcessor()
 	 * @param response HttpResponse from the EnsEMBL service
 	 * @param originalURI The URI of the EnsEMBL service queried (e.g rest.ensembl.org or rest.ensemblgenomes.org)
 	 * @return EnsemblServiceResult object containing the relevant information from the response
+	 * @deprecated use {@link #processResponse(HttpResponse) instead}
 	 */
+	@Deprecated
```

Comment:
Why would we keep code we won't be using anymore? I can imagine having it for a little while after, but in time wouldn't it make sense to remove it so the class doesn't get bloated?

Reply:
Usually if you're producing a public API, you mark old code as deprecated for a release or two before removing it. That way, users who see it's deprecated can switch to something else. We don't really have a lot of external users so it's probably safe to remove it immediately, or in the next release. But some `@Deprecated` things can linger for years. I think there are some in the core Java libraries. They aren't used or recommended, but they are left so as not to break anything that relies on them.

Reply:
Yeah, I'm trying to follow semantic versioning for our releases.  If we were to remove a public method instead of deprecating it, it would require moving release-common-lib to v2.0.  Actually, since the public API has new but backwards-compatible changes, I should update the version to 1.2.0 instead of 1.1.1.

I don't mind going to v2.0, but I thought maybe we should give it a bit of time to see if there are any more changes or additions to release-common-lib before incrementing the major version.

Reply:
Hmmm... I think 1.2.0 is appropriate since no new major functionality has been introduced, and it _really_ should be compatible with the code that's already using it.

Reply:
Yeah, for this PR I think I'm going to go with v.1.2.0: https://github.com/reactome/data-release-pipeline/pull/121/commits/59e5bb48da93b4caff70260fb86a5f2963cedc4a